includes/User.php

```diff
@@ -1141,6 +1155,23 @@ private function loadFromSession() {
 			return false;
 		}
 
+		if ( !$this->isUserAuthenticated() ) {
```

Comment:
should we remove the statements above these? seems like if we want Helios to be the only source of authentication we can't let other cases early-out